lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_methods.rb

```diff
@@ -7,16 +7,16 @@ class MiqAeServiceMethods
 
     SYNCHRONOUS = Rails.env.test?
 
-    def self.send_email(to, from, subject, body, content_type: nil, cc: nil, bcc: nil) # rubocop:disable Naming/MethodParameterName
+    def self.send_email(to, from, subject, body, options = {}) # rubocop:disable Naming/MethodParameterName
```

Comment:
This method needed to be changed to no longer accept kwargs as the caller in `execute_with_user` will append the kwargs to the incoming arguments as a hash so no kwargs will be sent.  I also need to change `miq_send_email` to no longer send kwargs as it bypassed `execute_with_user`.

Reply:
This is now stale, I figured out a way to treat this method differently and accept kwargs or options hashes so we don't need to break existing callers using kwargs.